src/object/obj_verify.c

```diff
@@ -134,10 +135,10 @@ dc_obj_verify_fetch(struct dc_obj_verify_args *dova)
 	dova->fetch_sgl.sg_nr_out = 1;
 	dova->fetch_sgl.sg_iovs = &dova->fetch_iov;
 
-	rc = dc_obj_fetch_shard_task_create(dova->oh, dova->th,
-		DIOF_TO_SPEC_SHARD, dc_obj_anchor2shard(&dova->dkey_anchor),
-		&cursor->dkey, 1, iod, &dova->fetch_sgl, NULL, NULL, NULL,
-		&task);
+	shard = dc_obj_anchor2shard(&dova->dkey_anchor);
+	rc = dc_obj_fetch_task_create(dova->oh, dova->th, 0, &cursor->dkey, 1,
+				      DIOF_TO_SPEC_SHARD, iod, &dova->fetch_sgl,
+				      NULL, &shard, NULL, NULL, &task);
```

Comment:
this case still use the ptr point to stack variable, although not a real problem for now as "dc_task_schedule(task, true)" below and current tse behavior. just a note for following refine if needed.

Reply:
This one is for verify object consistency, totally controlled by the DAOS client internal logic, always be sync mode.